main/modes/lumberjack/lumberjackGame.c

```diff
@@ -1496,6 +1505,11 @@ void baseMode(int64_t elapsedUs)
     if (NULL != lumv->ghost && lumv->ghost->active && lumv->gameState != LUMBERJACK_GAMESTATE_GAMEOVER)
     {
         lumv->ghost->timerFrameUpdate += elapsedUs;
+        if (lumv->ghost->attackTime > 0)
+        {
+            lumv->ghost->attackTime       -= elapsedUs/1000;
```

Comment:
Don't forget to `make format`!

Reply:
The make file wasn't working for me. So, I manually ran it in [8c9b799d].

That being said, I even tried using the instructions that jvegs mentioned and had the same results. So... I manually said "Format"